cmd/platform.go

```diff
@@ -18,7 +19,9 @@ func configurePlatform(cmd *cobra.Command) {
 	flags := cmd.Flags()
 
 	flags.StringP("base-url", "g", "", "Base URL of the (v3) GitHub API, needs to be changed if GitHub enterprise is used. Or the url to a self-hosted GitLab instance.")
-	flags.StringP("token", "T", "", "The GitHub/GitLab personal access token. Can also be set using the GITHUB_TOKEN/GITLAB_TOKEN environment variable.")
+	flags.BoolP("insecure", "I", false, "Insecure controls whether a client verifies the server certificate chain and host name. Used only for Bitbucket server.")
+	flags.StringP("username", "u", "", "The Bitbucket server username.")
```

Comment:
I've had no hard rules for when to have one-character flags. But I think that options that only work on one SCM should probably be excluded and only have the full option name.

Reply:
Agreed. Removed `I` from arg parser.